### DISCUSSION TITLE
Swin T 실행 에러

### DISCUSSION THREAD

**Discussion by JiyouSeo** [boostcampaitech2/object-detection-level2-cv-06#105] [Q&A] [answered]
https://github.com/shinya7y/UniverseNet 이 깃헙을 clone해서
Swin T Config를 돌려봤는데, mmdetection 버전 마다
mmdet/models/backbones/swin.py를 참고하는 경우도 있고, mmdet/models/backbones/swin_transformer.py를 참고하는 경우도 있어서
swin_transformer.py를 추가 해주었지만, 코드가 돌아가지 않는 슬픈 소식이 있습니다..
(참고링크 https://github.com/open-mmlab/mmdetection/issues/5392)
제 생각에 Soft teacher 처럼 env하나를 새로 만들어서 install 작업이 들어가야할 것 같습니다..
https://github.com/shinya7y/UniverseNet/blob/master/docs/get_started.md

Soft teacher 처럼 보류할까요? 아니면 제가 직접 custom해서 제출했었던 cascade_rcnn_swin_t config가 있긴한데,(#42) (리더보드 mAP : 0.522)
이걸 좀 더 발전시키는 방법도 있긴 합니다.

**Comment by sne12345** [comment, marked as answer]
KeyError: "CascadeRCNN: 'SwinTransformerOriginal is not in the models registry'"
이 에러인가요??

**Reply by sne12345** [reply]
저는
mmdet에 SwinTransformerOriginal model을 등록하고
pip3 install -v -e .
해서 해결했었습니다

**Reply by JiyouSeo** [reply]
감사드립니다 !
